### PR TITLE
[NG] Added the service mesh management section

### LINKED ISSUE #1420: Implementation of Service Mesh Management section in the homepage
**Current Behavior**
We currently have no implementation of the Service Mesh Management section. 

**Desired Situation**
The design of the home page is currently approved on figma, and we will appreciate someone jumping on the task of implementing it 

https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=1869%3A2967

Check out the design below

Group 1008

[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Harshita-Kanal**
Signed-off-by: Harshita-Kanal <harshita.kgv@gmail.com>

**Description**

This PR fixes #1420 

**Notes for Reviewers**
Implemented the section.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Hello! 😄

File: src/sections/service-mesh-management/ServiceMesh.style.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+       font-style: normal;
+       font-weight: 300;
+       font-size: 18px;
+       line-height: 27px;
+   }
+
+  @media only screen and (max-width: 480px) {
+    padding: 40px 0 60px 0;
+    .mesh-image{
+        display: none;
```
Comment (Jashpatel1) [inline src/sections/service-mesh-management/ServiceMesh.style.js:55]: Any specific reasons behind this?

Comment (Harshita-Kanal) [reply src/sections/service-mesh-management/ServiceMesh.style.js:55]: In the mobile view, the image should be visible or disappear?

Comment (Jashpatel1) [reply src/sections/service-mesh-management/ServiceMesh.style.js:55]: I think it should be visible.

**Comment by Harshita-Kanal** [comment]
Made image visible in mobile view, sorry for the force push, forgot to sign the commits 😅

**Comment by leecalcote** [comment]
Thank you, @Harshita-Kanal. 😄

**Pull request merged**